scripts/format-py-code

```diff
@@ -51,4 +51,4 @@ if [[ ${verbose} -eq 1 ]]; then
     args+=("--verbose")
 fi
 
-"${cmd[@]}" "${args[@]}" "$this_dir/.."
+"${cmd[@]}" "${args[@]}" "--exclude" 'ltp/|lkl/|openenclave/|host-musl/|sgx-lkl-musl/|build/|build_musl/|third_party/' "$this_dir/.."
```

Comment:
This is not the right place. Please look at https://github.com/lsds/sgx-lkl/blob/oe_port/pyproject.toml

Reply:
I think excluding the .git directory in the LTP submodule would make it significantly quicker

Reply:
@paulcallen I am already excluding ltp folder and which means it will skip any subfolder as well.

Reply:
@letmaik I moved ltp exclusion to pyproject.toml file. I wasn't aware of such a file. Done.